src/Tools/dotnet-counters/Exporters/JSONExporter.cs

```diff
@@ -72,11 +72,11 @@ public void CounterPayloadReceived(CounterPayload payload, bool _)
                     builder.Clear();
                 }
                 builder
-                    .Append("{ \"timestamp\": \"").Append(DateTime.Now.ToString("u")).Append("\", ")
-                    .Append(" \"provider\": \"").Append(JsonEscape(payload.Provider)).Append("\", ")
-                    .Append(" \"name\": \"").Append(JsonEscape(payload.GetDisplay())).Append("\", ")
-                    .Append(" \"tags\": \"").Append(JsonEscape(payload.Metadata)).Append("\", ")
-                    .Append(" \"counterType\": \"").Append(JsonEscape(payload.CounterType.ToString())).Append("\", ")
+                    .Append("{ \"timestamp\": \"").Append(DateTime.Now.ToString("O")).Append("\", ")
+                    .Append(" \"provider\": \"").Append(JsonEscape(payload.ProviderName)).Append("\", ")
```

Comment:
How does this change the output of the other fields?

Reply:
I think that I may have merged some additional changes, because I know that what I needed to do was to change the U to an O. I am still trying to figure out if/why I did the rest of the changes.

Reply:
I think something happened in the time that my PR was sitting, so I am going to need to redo it because of the changes that have been made.

Reply:
Here is an example of what the output looks like

Timestamp,Provider,Counter Name,Counter Type,Mean/Increment
11/8/2023 9:37:32 AM,System.Runtime,CPU Usage (%),Metric,0
11/8/2023 9:37:56 AM,System.Runtime,CPU Usage (%),Metric,0
11/8/2023 9:37:32 AM,System.Runtime,Working Set (MB),Metric,72
11/8/2023 9:37:32 AM,System.Runtime,GC Heap Size (MB),Metric,2
11/8/2023 9:37:32 AM,System.Runtime,Gen 0 GC Count (Count / 1 sec),Rate,0
11/8/2023 9:37:32 AM,System.Runtime,Gen 1 GC Count (Count / 1 sec),Rate,0
11/8/2023 9:37:32 AM,System.Runtime,Gen 2 GC Count (Count / 1 sec),Rate,0
11/8/2023 9:37:32 AM,System.Runtime,ThreadPool Thread Count,Metric,1
11/8/2023 9:37:32 AM,System.Runtime,Monitor Lock Contention Count (Count / 1 sec),Rate,0
11/8/2023 9:37:32 AM,System.Runtime,ThreadPool Queue Length,Metric,0
11/8/2023 9:37:32 AM,System.Runtime,ThreadPool Completed Work Item Count (Count / 1 sec),Rate,0

Reply:
I'd reset the branch and add only the one change - I don't think it makes sense it shows a diff.